### PR TITLE
fix #108. Add improved About page.

### LINKED ISSUE #108: About page

### PR DESCRIPTION
**Pull request by SebastianMygind**
This pull requests adds an improved About page that revolves around "AboutCards" which include a picture and some self chosen information about all of us, i have already created myself as an example. To add the missing information send me a message about what should be added or suggest changes through github.
sry for possible spelling mistakes in your names.

In the about cards i use a MUI component called CardActionArea. Currently it does not have any use, but i would like to add an overlay when you click that displays some contact info

### CODE REVIEW

File: components/about-card.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import type { FC, ReactElement } from "react";
+import Grid from "@mui/material/Grid";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import { CardActionArea } from "@mui/material";
```
Comment (audio-engineer) [inline components/about-card.tsx:8]:
```suggestion
import CardActionArea from "@mui/material/CardActionArea";
```

File: components/about-card.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
```
Comment (audio-engineer) [inline components/about-card.tsx:1]: This does what?

File: components/about-card.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  image_name,
+  name,
+  description,
+}: AboutUsProps): ReactElement | null => {
+  const ALT_TEXT = `picture of ${name}`;
+  const image = `/images/${image_name}`;
+
+  return (
+    <Grid item>
+      <Card sx={{ width: CARD_WIDTH, boxShadow: 1 }}>
```
Comment (audio-engineer) [inline components/about-card.tsx:29]: Ah, so developers have to scroll up to see the width. So convenient...

File: components/about-card.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        <CardActionArea>
+          <CardMedia
+            component="img"
+            height={CARD_LENGTH}
```
Comment (audio-engineer) [inline components/about-card.tsx:33]: and here

File: components/about-card.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            image={image}
+            alt={ALT_TEXT}
```
Comment (audio-engineer) [inline components/about-card.tsx:35]: and here

File: components/about-card.tsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+import type { FC, ReactElement } from "react";
+import Grid from "@mui/material/Grid";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import { CardActionArea } from "@mui/material";
+
+const CARD_WIDTH = 250;
+const CARD_LENGTH = 200;
```
Comment (audio-engineer) [inline components/about-card.tsx:11]: Bad variable case. In fact, completely unnecessary variables

File: components/about-card.tsx [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+interface AboutUsProps {
+  readonly image_name: string;
```
Comment (audio-engineer) [inline components/about-card.tsx:14]: Bad case

File: components/about-card.tsx [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import { CardActionArea } from "@mui/material";
+
+const CARD_WIDTH = 250;
+const CARD_LENGTH = 200;
+
+interface AboutUsProps {
```
Comment (audio-engineer) [inline components/about-card.tsx:13]: Bad naming

File: components/about-page.tsx [hunk 9]
```diff
@@ -1,11 +1,52 @@
...
+        chess web app could be created, and to experiment with teaching
+        methodology. To enhance the teaching process, they combined the power of
+        chess engines with the social experience of player-versus-player
+        gameplay.
+      </Typography>
+      <Grid container spacing={2} justifyContent="center">
+        <AboutCard
+          image_name="sebastian.jpg"
+          name="Sebastian Mygind"
+          description="“Sebastian Mygind, a 21-year-old computer
```
Comment (audio-engineer) [inline components/about-page.tsx:24]: You didn't listen up during the meeting on the 12th of April

File: components/about-card.tsx [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+  readonly image_name: string;
+  readonly name: string;
+  readonly description: string;
```
Comment (audio-engineer) [inline components/about-card.tsx:16]: See comment for usage

File: components/about-card.tsx [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+const AboutCard: FC<AboutUsProps> = ({
+  image_name,
+  name,
+  description,
+}: AboutUsProps): ReactElement | null => {
+  const ALT_TEXT = `picture of ${name}`;
+  const image = `/images/${image_name}`;
```
Comment (audio-engineer) [inline components/about-card.tsx:25]: Bad naming and case. What's the difference between the two variables?

**Review by audio-engineer — CHANGES_REQUESTED** [review]

**Review by audio-engineer — APPROVED** [review]

**Review by Danar435 — APPROVED** [review]

**Pull request merged**